src/components/mobile/MobileMenu.tsx

```
@@ -0,0 +1,59 @@
+    return () => document.removeEventListener('keydown', handleEsc);
+  }, []);
+
+  return (
+    <div className={className}>
+      {/* Mobile menu trigger */}
+      <button
+        onClick={() => setIsOpen(!isOpen)}
+        className="md:hidden fixed top-4 left-4 z-50 p-2 bg-gray-900 text-white rounded-lg shadow-lg"
+        aria-label="Toggle navigation menu"
+        aria-expanded={isOpen}
+      >
```

Biome (2.3.14)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)